medic/medic-downloadlogs.js

```diff
@@ -0,0 +1,80 @@
+var path = require('path'),
+    fs = require('fs'),
+    https = require('https'),
+    optimist = require('optimist');
+    q = require('Q'),
+    mkdirp = require('mkdirp')
+    util = require('util');
+
+var SERVER = "https://ci.apache.org";
+var BUILDERS = ["cordova-windows-store8.1", "cordova-ios", "cordova-windows-phone8.1", "cordova-android-osx","cordova-android-win"];
+//var BUILDERS = ["cordova-windows-store8.1"];
+var STEPS = ["running-tests", "gathering-logs", "getting-test-results"];
+
+function downloadLogs(outputDir) {
+    var counter = 0;
+    var builderPromises = BUILDERS.map(function(builder) {
```

Comment:
It can be - but I prefer map :)

---
In reply to: [59606984](https://github.com/apache/cordova-medic/pull/87#discussion_r59606984) [](ancestors = 59606984)